dm/tests/openapi/client/openapi_task_check

```diff
@@ -221,32 +267,6 @@ def get_task_list_with_status(task_count, task_name, status_count):
     assert find_task
 
 
-def pause_task_success(task_name, source_name):
```

Comment:
Have replaced pause and resume by start and stop?

Reply:
yes, after this pr there is not pause/resume in openapi, just start/stop